x/gov/client/cli/tx.go

```diff
@@ -26,8 +26,8 @@ const (
 	FlagDescription = "description"
 	// Deprecated: only used for v1beta1 legacy proposals.
 	FlagProposalType = "type"
-	// Deprecated: only used for v1beta1 legacy proposals.
 	FlagDeposit   = "deposit"
+	FlagMetadata  = "metadata"
```

Comment:
```suggestion
```

is this used somewhere?

Reply:
Don't think so. I've removed it. Thanks for the spot